cloudinit/config/cc_ca_certs.py

```diff
@@ -5,48 +5,69 @@
 """CA Certs: Add ca certificates."""
 
 import os
+import shutil
 from logging import Logger
 from textwrap import dedent
 
+from cloudinit import log as logging
 from cloudinit import subp, util
 from cloudinit.cloud import Cloud
 from cloudinit.config import Config
 from cloudinit.config.schema import MetaSchema, get_meta_doc
 from cloudinit.settings import PER_INSTANCE
 
+LOG = logging.getLogger(__name__)
+
 DEFAULT_CONFIG = {
-    "ca_cert_path": "/usr/share/ca-certificates/",
-    "ca_cert_filename": "cloud-init-ca-certs.crt",
+    "ca_cert_path": None,
+    "ca_cert_blacklist_path": None,
+    "ca_cert_local_path": "/usr/local/share/ca-certificates/",
+    "ca_cert_filename": "cloud-init-ca-cert-%(number)s.crt",
     "ca_cert_config": "/etc/ca-certificates.conf",
-    "ca_cert_system_path": "/etc/ssl/certs/",
     "ca_cert_update_cmd": ["update-ca-certificates"],
 }
 DISTRO_OVERRIDES = {
+    "fedora": {
+        "ca_cert_path": "/etc/pki/ca-trust/",
+        "ca_cert_blacklist_path": None,
+        "ca_cert_local_path": "/usr/share/pki/ca-trust-source/anchors/",
+        "ca_cert_filename": "cloud-init-ca-cert-%(number)s.crt",
+        "ca_cert_config": None,
+        "ca_cert_update_cmd": ["update-ca-trust"],
+    },
+    "freebsd": {
+        "ca_cert_path": "/usr/share/certs/trusted/",
+        "ca_cert_blacklist_path": "/usr/share/certs/blacklisted/",
```

Comment:
This is different between 13 and 14. In 14 it's `/usr/share/certs/untrusted`
We need to find a good way to map this

Reply:
Removed FreeBSD-related changes.